### ISSUE TITLE
The box don't follow the input

### ISSUE THREAD

**Issue by metalfox6383** [Pixabay/jQuery-autoComplete#25]
Hi:

I like your plugin. I would like the box follow the input when the page is in scrolling. That is to say, when the box is visible and I scroll the web, the box stay in the same position until I press some key. Is it possible?

Thanks.

**Comment by SimonSteinberger** [comment]
It does follow on the demo page - at least on my test system. How does the demo behave on your end?

**Comment by metalfox6383** [comment]
Yeah. You right. Sorry. Now I see the problem. I use bootstrap and I have a form in a modal. The problem exists when I use your plugin in a input in a modal. I will seek a solution and I will comment. Thanks.

**Comment by SimonSteinberger** [comment]
Cool, thanks :)
At least it's a rather rare scenario.

**Comment by metalfox6383** [comment]
Hi my friend. I didn't find a solution. I'm sorry. Your element may be to need to be father's son of the input text. Your component is all right and that detail doesn't mater by now. Greetings.

**Comment by friarpat** [comment]
I think I have a variation on this theme.

I have a div above the textbox that the autocomplete is attached to that presents errors to the user.

div   potential error text   div
div 
     textbox where the autoComplete is attached to
div

  Normally the error div is obviously not displayed  (display:none).   If the autoComplete is presenting a list and an error is presented, the error div is displayed with the appropriate content - but the autoComplete doesn't "slide" down with the textbox it is attached to.    

Is there a way to refresh the autoComplete in this situation?  I can obviously close the autoComplete when an error comes in,  but it would be nice to just close it and force it to reopen on the textbox at its new location.

Thanks!

**Comment by ErnestBrandi** [comment]
```javascript
$("#your-modal").scroll(function (event) {
    $(".autocomplete-suggestions").css("display", "none");
});
```

**Comment by mbraz** [comment]
Hi guys!

Someone discover a solution or a substitute plugin?

Thanks
